compiler/src/iree/compiler/Codegen/Common/GPU/test/reduce_bank_conflicts.mlir

```diff
@@ -4,17 +4,17 @@
 
 // CHECK-LABEL: func.func @pad_alloc
 func.func @pad_alloc(%a: memref<1024x1024xf32>) {
-// CHECK: %[[A:.*]] = memref.alloc() : memref<4x32x68xf32, #gpu.address_space<workgroup>>
+// CHECK: %[[A:.*]] = memref.alloc() : memref<4x32x66xf32, #gpu.address_space<workgroup>>
   %0 = memref.alloc() : memref<4x32x64xf32, #gpu.address_space<workgroup>>
-// CHECK: %[[S1:.*]] = memref.subview %[[A]][0, 0, 0] [4, 32, 64] [1, 1, 1] : memref<4x32x68xf32, #gpu.address_space<workgroup>> to memref<4x32x64xf32, strided<[2176, 68, 1]>, #gpu.address_space<workgroup>>
-// CHECK: %[[S2:.*]] = memref.subview %[[S1]][0, 0, 0] [1, 32, 64] [1, 1, 1] : memref<4x32x64xf32, strided<[2176, 68, 1]>, #gpu.address_space<workgroup>> to memref<1x32x64xf32, strided<[2176, 68, 1]>, #gpu.address_space<workgroup>>
+// CHECK: %[[S1:.*]] = memref.subview %[[A]][0, 0, 0] [4, 32, 64] [1, 1, 1] : memref<4x32x66xf32, #gpu.address_space<workgroup>> to memref<4x32x64xf32, strided<[2112, 66, 1]>, #gpu.address_space<workgroup>>
```

Comment:
Nit: I personally hate this interspersed check lines style.

Reply:
+1, I prefer check at the end also.